### PR TITLE
Rename rapids-js to node-rapids

### PR DESCRIPTION
**Pull request by ajschmidt8**
This PR replaces all the references to `rapids-js` with `node-rapids`.

### CODE REVIEW

File: modules/cugraph/README.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
-# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp; rapids-js cuGraph - GPU Graph Analytics
+# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp; node-rapids cuGraph - GPU Graph Analytics
 
 The js bindings for [cuGraph](https://github.com/rapidsai/cugraph) is a collection of GPU accelerated graph algorithms that process data found in GPU DataFrames.
 
-For detailed cuGraph-js API, follow our [API Documentation](https://rapidsai.github.io/rapids-js/modules/cugraph_src.html).
+For detailed cuGraph-js API, follow our [API Documentation](https://rapidsai.github.io/node-rapids/modules/cugraph_src.html).
```
Comment (AjayThorve) [inline modules/cugraph/README.md:5]:
```suggestion
For detailed node-cuGraph API, follow our [API Documentation](https://rapidsai.github.io/node-rapids/modules/cugraph_src.html).
```

File: modules/cudf/README.md [hunk 2]
```diff
@@ -1,39 +1,49 @@
...
-console.log(...groups['keys'].get('a')); // [0,1,2,3,4,5]
-console.log(...groups.values?.get('b')); // [2,2,1,1,0,0]
-console.log(...groups['offsets']); // [0,1,2,3,4,5,6]
+console.log(...groups["keys"].get("a")); // [0,1,2,3,4,5]
+console.log(...groups.values?.get("b")); // [2,2,1,1,0,0]
+console.log(...groups["offsets"]); // [0,1,2,3,4,5,6]
 ```
 
-For detailed cudf-js API, follow our [API Documentation](https://rapidsai.github.io/rapids-js/modules/cudf_src.html).
+For detailed cudf-js API, follow our [API Documentation](https://rapidsai.github.io/node-rapids/modules/cudf_src.html).
```
Comment (AjayThorve) [inline modules/cudf/README.md:39]:
```suggestion
For detailed node-cuDF API, follow our [API Documentation](https://rapidsai.github.io/node-rapids/modules/cudf_src.html).
```

File: modules/cuspatial/README.md [hunk 3]
```diff
@@ -1,14 +1,15 @@
-# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp; rapids-js cuSpatial - GPU-Accelerated Spatial and Trajectory Data Management and Analytics Library
+# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp; node-rapids cuSpatial - GPU-Accelerated Spatial and Trajectory Data Management and Analytics Library
 
 JS bindings for [cuSpatial](https://github.com/rapidsai/cuspatial).
 
-For detailed cuSpatial-js API, follow our [API Documentation](https://rapidsai.github.io/rapids-js/modules/cuspatial_src.html).
+For detailed cuSpatial-js API, follow our [API Documentation](https://rapidsai.github.io/node-rapids/modules/cuspatial_src.html).
```
Comment (AjayThorve) [inline modules/cuspatial/README.md:5]:
```suggestion
For detailed node-cuSpatial API, follow our [API Documentation](https://rapidsai.github.io/node-rapids/modules/cuspatial_src.html).
```

**Review by AjayThorve — COMMENTED** [review]
LGTM, just a couple of changes to some modules' readme

**Review by AjayThorve — APPROVED** [review]

**Review by trxcllnt — APPROVED** [review]

**Pull request merged**